libs/eslint-plugin-vx/src/rules/gts_spread_like_types.ts

```diff
@@ -115,10 +114,15 @@ const rule: TSESLint.RuleModule<
           }
 
           /* istanbul ignore next - this should not be possible */
-          default:
+          default: {
+            // eslint-disable-next-line no-underscore-dangle, @typescript-eslint/no-unused-vars
+            const _exhaustiveCheck: never = node.parent;
```

Comment:
I want `throwIllegalValue`. I guess I could add it here as the 3rd place in the codebase it exists…

Reply:
if you'd like. i'm also fine with this